docs/developer-docs/multi-chain/chain-key-tokens/ckerc20/using-cketh-in-dapps.mdx

```diff
@@ -0,0 +1,21 @@
+---
+keywords: [advanced, ethereum, eth, cketh, chain-key ethereum]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Using ckERC20 tokens in dapps
+
+<MarkdownChipRow labels={["Advanced", "Ethereum" ]} />
+
+## Overview
+
+All ckERC20 tokens are ICRC-2 compliant tokens, which means that they support the [ICRC-1](https://github.com/dfinity/ICRC-1/blob/main/standards/ICRC-1/README.md) and [ICRC-2](https://github.com/dfinity/ICRC-1/blob/main/standards/ICRC-2/README.md) token standards.
+
+To integrate ckERC20 tokens into your dapp, you can simply write code that uses the [ICRC-1](https://github.com/dfinity/ICRC-1/blob/main/standards/ICRC-1/README.md) and [ICRC-2](https://github.com/dfinity/ICRC-1/blob/main/standards/ICRC-2/README.md) standards. For token transfers and account balances, requests must be sent to the corresponding ckERC20 ledger canister.
+
+## Write a smart contract that uses ckERC20 tokens
+
+The same code as provided for [ckBTC](/docs/current/developer-docs/multi-chain/chain-key-tokens/ckbtc/using-ckbtc-in-dapps#write-a-smart-contract-that-uses-ckbtc) and [ckETH](/docs/current/developer-docs/multi-chain/chain-key-tokens/cketh/using-cketh-in-dapps#write-a-smart-contract-that-uses-cketh) can be used, the only difference being the targeted ledger canister.
```

Comment:
Should we include an example showing the ckUSDC ledger canister as the targeted canister?

Reply:
Good idea. Done!